ctk/features/flow.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Flow Directive
+  As an implementer of the workflow DSL
+  I want to ensure that tasks are executed in the correct order
+  So that my implementation conforms to the expected behavior
+
+  Scenario: Implicit Sequence Flow
+    Given a workflow with definition:
+    """yaml
+    document:
+      dsl: 1.0.0-alpha1
+      namespace: default
+      name: implicit-sequence
+    do:
+      setRed:
+        set:
+          colors: '${ .colors + [ "red" ] }'
+      setGreen:
+        set:
+          colors: '${ .colors + [ "green" ] }'
+      setBlue:
+        set:
+          colors: '${ .colors + [ "blue" ] }'
+    """
+    When the workflow is executed
+    Then the workflow should complete with output:
+    """yaml
+    colors: [ red, green, blue ]
+    """
+    And setRed should run first
+    And setGreen should run after setRed
+    And setBlue should run after setGreen
+
+  Scenario: Explicit Sequence Flow
+    Given a workflow with definition:
+    """yaml
+    document:
+      dsl: 1.0.0-alpha1
+      namespace: default
+      name: explicit-sequence
+    do:
+      setRed:
+        set:
+          colors: '${ .colors + [ "red" ] }'
```

Comment:
As discussed in slack, I believe it will be easier to write `colors: .colors +  [ "red "],` if we assume that by default string values are always expressions.

Reply:
I thought a lot about it, and I think it's a very oppiniated matter, from both ends. As discussed in slack, I do think your proposal makes a LOT of sense. I also believe, especially because it is opiniated, that the matter is quite subtle, and could be addressed in a countless amount of ways.

As a matter of consequence, I propose we open an issue addressing this specific subject and solve it for `alpha2`, so we don't diverge as of now on what was being done regarding that. WDYT?

Reply:
Sure I will open an issue with the different proposals

Reply:
https://github.com/serverlessworkflow/specification/issues/848